cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -754,38 +754,131 @@
 
   <property>
     <name>kafka.default.replication.factor</name>
+    <value>${kafka.server.default.replication.factor}</value>
+    <description>
+      CDAP Kafka service replication factor (deprecated: replaced with kafka.server.replication.factor)
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.log.retention.hours</name>
+    <value>${kafka.server.log.retention.hours}</value>
+    <description>
+      The number of hours to keep a log file before deleting it (deprecated: replaced with
+      kafka.server.log.retention.hours)
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.num.partitions</name>
+    <value>${kafka.server.num.partitions}</value>
+    <description>
+      Default number of partitions for a topic
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.seed.brokers</name>
+    <value>127.0.0.1:9092</value>
+    <description>
+      Comma-separated list of CDAP Kafka service brokers; for distributed CDAP,
+      replace with list of FQDN:port brokers (deprecated: replaced with kafka.server.seed.brokers)
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.server.default.replication.factor</name>
     <value>1</value>
     <description>
       CDAP Kafka service replication factor; used to replicate Kafka messages across
       multiple machines to prevent data loss in the event of a hardware
       failure. The recommended setting is to run at least two CDAP Kafka servers.
       If you are running two CDAP Kafka servers, set this value to 2; otherwise,
-      set it to the number of CDAP Kafka servers.
+      set it to the maximum number of tolerated machine failures
+      plus one (assuming you have that number of machines).
     </description>
   </property>
 
   <property>
-    <name>kafka.log.dir</name>
```

Comment:
You also need to deprecate the old one `kafka.log.dir` since it is a public config that user can override.